### DISCUSSION TITLE
Is an exported CABLES patch initialising in the global scope?

### DISCUSSION THREAD

**Discussion by cristianvogel** [cables-gl/cables_docs#529] [Answers & Questions] [answered]
I ask this, because I have a bug in my website project, that embeds a Cables patch. I am using a library called Elementary Audio, and it went up in version ( and went fully OSS btw ) https://github.com/elemaudio/elementary  

Anyway, in the v2.0.0 change, the Elementary core is not firing it's load() emit and we are trying to debug the behaviour. It would help to narrow things down a bit, if I could understand if there is something from the  CABLES patch object I need to be aware of, when initialising the other deps.

**Comment by steam0r** [comment]
hm, not sure i understand your problem. are there any specific errors or is just the load event not fired (which then would be on the libraries side, i guess). would be helpful if you could provide a website with the problem and i'd be happy to have a look.

are you trying to load this library in an op? outside of cables? customop? userop?

**Reply by cristianvogel** [reply]
Hey thanks for the response and the offer for help , much appreciated - I am going to put a few  more hours into the debugging and then reach out after that with what I have learned.

**Comment by cristianvogel** [comment, marked as answer]
I have ruled out a conflict with Cables.  

// _are you trying to load this library in an op? outside of cables? customop? userop?_

Actually, no I am embedding an exported patch.js
